api/candlepin-api-spec.yaml

```diff
@@ -232,6 +232,61 @@ components:
             manifest:
               type: boolean
 
+    AsyncJobStatusDTO:
+      description: DTO representing an async job status
+      discriminator:
+        propertyName: type
```

Comment:
Is this needed? From what I recall there is a conversation in another PR between @abhiskum and @Ceiu that we should not use discriminator.

Reply:
@nikosmoum yes, we have decided not to use 'discriminator' in DTO definition.

Reply:
I missed PR conversion.
After a discussion with @abhiskum, I understood we should not be adding this. Now eliminated it from the code.